lib/ui/painting/vertices.cc

```diff
@@ -77,10 +82,14 @@ void Vertices::init(SkVertices::VertexMode vertex_mode,
     DecodeInts<SkColor>(colors, builder.colors());
   }
 
-  if (indices.data())
-    DecodeInts<uint16_t>(indices, builder.indices());
+  if (indices.data()) {
+    std::copy(indices.data(), indices.data() + indices.num_elements(),
+              builder.indices());
+  }
```

Comment:
Return `false` if the data is not present?

Reply:
`indices` is an optional argument